### ISSUE TITLE
Should this repo be archived? And migrate open issues to GitHub Discussions?

### ISSUE THREAD

**Issue by oscard0m** [octokit/discussions#28]

**Comment by gr2m** [comment]
This repository is meant as place to discuss topics for all of the Octokit libraries: Ruby, .net, JS, and other ones in future. So I would leave the repository, but we could enable discussions, once a need arises

**Comment by oscard0m** [comment]
> This repository is meant as place to discuss topics for all of the Octokit libraries: Ruby, .net, JS, and other ones in future. So I would leave the repository, but

Has sense to keep it alive then, yep.

> we could enable discussions, once a need arises

Don't you prefer GitHub Discussions for discussions instead of issues which usually are related to work to do?

**Comment by gr2m** [comment]
Yeah we can move all open issues to discussions and disable issues. There is no urgency though, we don't use this repository at all right now
